pkg/kubernetes/deployment/extnv1beta1/v1alpha1/deployment.go

```diff
@@ -17,9 +17,19 @@ limitations under the License.
 package v1alpha1
 
 import (
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	stringer "github.com/openebs/maya/pkg/apis/stringer/v1alpha1"
 	"github.com/pkg/errors"
+	"k8s.io/api/core/v1"
 	extnv1beta1 "k8s.io/api/extensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// StoragePoolKindCSPC holds the value of CStorPoolCluster
+	StoragePoolKindCSPC = "CStorPoolCluster"
```

Comment:
we are pushing cspc related things to deployment which is very generic.
we need to find a proper home for this constant

Reply:
fixing this comment avoids the extra import

Reply:
Yes -- will fix it. Any way -- Not going to use `extnv1beta1` .. The changes are made bu will revert. 
Also -- what extra import will be avoided ?